kore/app/share/GlobalMain.hs

```diff
@@ -478,13 +484,19 @@ type LoadedModule = VerifiedModule Attribute.Symbol
 
 type LoadedDefinition = (Map ModuleName LoadedModule, Map Text AstLocation)
```

Comment:
I think we should make this a record data type and embed the `KFileLocations` there.